docs/01-module-block-1/02-learning-goals.adoc

```diff
@@ -1,25 +1,44 @@
 === {learning-goals}
 
 // tag::DE[]
-Die Teilnehmer:Innen …
+In diesem Kapitel erreichen die Teilnehmer:innen die folgenden Lernziele:
 
 [[LZ-1-1]]
-==== LZ 1-1: Wissen wie man Künstliche Intelligenz definiert und die Anordnung zu Machine Learning, Data Science, Deep Learning, Generative AI.
+==== LZ 1-1: Definition Künstliche Intelligenz inklusive der Einordnung von Machine Learning, Data Science, Deep Learning, Generative AI
+
+Die Teilnehmer:innen wissen wie man Künstliche Intelligenz definiert und wie Machine Learning, Data Science, Deep Learning, Generative AI darin eingeordnet werden.
 
 [[LZ-1-2]]
-==== LZ 1-2: Wissen, wie man typische allgemeine Use-Cases für KI spezifiziert, z.B. Bilderkennung & -erzeugung, Sprachverarbeitung, Vorhersagen, Personalisierung und Anomalieerkennung.
+==== LZ 1-2: Spezifikation typischer allgemeiner Anwendungsfälle für KI
```

Comment:
Wir haben in einigen der anderen Lehrpläne die Lernziele als Sätze mit Verben definiert. 
Hier am Beispiel wäre das 

> Typische allgemeine Anwendungsfälle für KI spezifizieren

Gefällt mir persönlich besser, da sonst evtl. Leser:innen das so auffassen könnten, als würden hier die Anwendungsfälle spezifiziert, nicht, dass sie lernen, wie es geht. Was denkst du darüber?

Reply:
Ja, ok, kann ich ändern, ich hatte euren Änderungswunsch letztes Mal so verstanden, dass ihr es hier eben nicht wollt und dass das Verb dann jeweils im Satz der Beschreibung unten steht. Das mag mein Missverständnis sein, lass uns darüber sprechen.